src/web/app/components/question-types/question-edit-details-form/mcq-question-edit-details-form.component.ts

```diff
@@ -22,6 +19,13 @@ export class McqQuestionEditDetailsFormComponent
     FeedbackParticipantType.STUDENTS_EXCLUDING_SELF, FeedbackParticipantType.TEAMS,
     FeedbackParticipantType.TEAMS_EXCLUDING_SELF, FeedbackParticipantType.INSTRUCTORS];
 
+  // Used to store and restore user input when user toggles generate option
+  storageModel: FeedbackMcqQuestionDetails = {
```

Comment:
Is there any better name than `storageModel`?

Reply:
`tempModel`? `cacheModel`? I can't really find a good name for it, therefore the long comment.